capa/main.py

```diff
@@ -71,14 +74,26 @@
     FORMAT_ELF,
     OS_WINDOWS,
     FORMAT_AUTO,
+    FORMAT_CAPE,
     FORMAT_SC32,
     FORMAT_SC64,
     FORMAT_DOTNET,
     FORMAT_FREEZE,
     FORMAT_RESULT,
 )
 from capa.features.address import NO_ADDRESS, Address
-from capa.features.extractors.base_extractor import BBHandle, InsnHandle, FunctionHandle, FeatureExtractor
+from capa.features.extractors.base_extractor import (
+    BBHandle,
+    CallHandle,
+    InsnHandle,
+    SampleHashes,
+    ThreadHandle,
+    ProcessHandle,
+    FunctionHandle,
+    FeatureExtractor,
+    StaticFeatureExtractor,
+    DynamicFeatureExtractor,
+)
 
 RULES_PATH_DEFAULT_STRING = "(embedded rules)"
 SIGNATURES_PATH_DEFAULT_STRING = "(embedded signatures)"
```

Comment:
- [ ] don't load signatures for dynamic analysis

Reply:
isn't this already being done with this code segment?
![image](https://github.com/mandiant/capa/assets/16624109/10b54987-e167-4134-b4bf-14ab5280c877)

if the context is dynamic analysis then the signatures array will be empty, should we try to remove that concept all together when the analysis is dynamic?

Reply:
Yes, there's still the output from `handle_common_args` though regardless of using them later or not
![image](https://github.com/mandiant/capa/assets/17606537/df53ad7f-599e-475c-bef5-73b96dc3e815)

Reply:
this is still an issue and I didn't see a quick and clean way to address it
yacine updated in https://github.com/mandiant/capa/pull/1697/commits/ffe6ab68428ce89d07dfbbe11509fb21e18befaa
but then it was removed again in https://github.com/mandiant/capa/pull/1697/commits/8ee97acf2a8904570be332065ea2ab37ff1d5e0f and https://github.com/mandiant/capa/pull/1697/commits/547502051fd109c71f2180f61f011aa0ccc69a89

Reply:
sorry, i reverted the changes since its causes tests to fail. i think a better fix will be to move the warning out of `handle_common_args`, though then we have to find all the right places to do it, and do it consistently.